packages/react-tooltip/src/components/Tooltip/useTooltip.tsx

```diff
@@ -203,16 +203,20 @@ export const useTooltip_unstable = (props: TooltipProps): TooltipState => {
   }
 
   if (relationship === 'label') {
-    // aria-label only works if the content is a string. Otherwise, need to use aria-labelledby.
-    if (typeof state.content.children === 'string') {
-      triggerProps['aria-label'] = state.content.children;
-    } else if (!isServerSideRender) {
-      triggerProps['aria-labelledby'] = state.content.id;
-      // Always render the tooltip even if hidden, so that aria-labelledby refers to a valid element
-      state.shouldRenderTooltip = true;
+    const hasLabel = child?.props && ('aria-label' in child.props || 'aria-labelledby' in child.props);
```

Comment:
Since `aria-labelledby` and `aria-describedby` can be a list of IDs do we need to handle these attributes differently when they are set and do something like append the tooltip ID?

A slightly contrived example:

```tsx
// Example JSX
<Tooltip content={<div>Additional tooltip information</div>} relationship="label">
    <Button aria-labelledby="default-label">...</Button>
</Tooltip>
<div id="default-label">Default Button Label/div>

// Should both IDs be present in rendered HTML?
<button aria-labelledby="default-label tooltip-randomly-generated-id">...</button>
```

Reply:
I don't think that's what someone would expect to happen, and I don't think it's generally safe to concatenate the label/labelledby like that. We don't know what the strings are, and if it makes sense to join them (or in what order, if so). If someone wanted to do that, they'd have to do that manually.